src/containers/shared/components/Transaction/OracleSet/parser.ts

```diff
@@ -0,0 +1,36 @@
+import { convertHexToString } from '../../../../../rippled/lib/utils'
+import { OracleSet } from './types'
+
+export function convertScaledPrice(assetPrice: string, scale: number) {
```

Comment:
Can you add a comment with the basic equation here so it's easier to read in the future? e.g. `assetPrice / 10**`scale`